CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # App Center SDK for iOS, macOS and tvOS Change Log
 
-## Version 4.2.1 (Under development)
+## Version 4.3.0 (Under development)
```

Comment:
Don't forget to change it to 4.3.1, after it is updated after the current release.

Reply:
It will be changed after merging these changes to develop.